shared/test/showcase/test_helper.rb

```diff
@@ -41,6 +44,27 @@ def generate_library_for_test imports, protos
   client_lib
 end
 
+def gapic_showcase_running?
+  system("ps aux | grep 'gapic-showcase run' | grep -v grep > /dev/null")
+end
+
+def tar_file_name
+  file_name = "gapic-showcase-#{GAPIC_SHOWCASE_VERSION}-"
+  case RUBY_PLATFORM
+  when /x86_64-linux/
+    file_name += "linux-amd64.tar.gz"
+  when /x86_64-darwin-\d+/
+    file_name += "darwin-amd64.tar.gz"
+  when /arm-linux/
+    file_name += "linux-arm.tar.gz"
+  when /arm64-darwin-\d+/
```

Comment:
On my Apple Silicon Mac, `RUBY_PLATFORM` is set to `"arm64-darwin23"`. So this regex at least needs to be fixed. It's possible the intel darwin regex is also similarly incorrect.

Reply:
Thanks, that was my mistake adding the extra dash. Checked on my mac too (2019 model): `universal.x86_64-darwin23`. I feel like the other ones should be good now too.